src/components/profile/ProfileContainer.jsx

```diff
@@ -1,16 +1,23 @@
 import { ReactComponent as IconPlus } from 'assets/icon_plus.svg';
 import { ReactComponent as SVGProfile } from 'assets/img_profile_small.svg';
+import useAPI from 'cores/hooks/useAPI';
 import React from 'react';
 import styled from 'styled-components';
 import { theme } from 'styles/theme';
 function ProfileContainer() {
+  const result = useAPI({
+    method: 'get',
+    url: `/profile/6290810aafae9f02409bdf47`,
+  });
```

Comment:
```suggestion
  const { data } = useAPI({
    method: 'get',
    url: `/profile/6290810aafae9f02409bdf47`,
  });
```
이렇게 구조분해 할당으로 받으면 밑에서 result.data.name 라고 할 필요 없이 data.name 이런식으로 바로 들어갈 수 있어!!!